### DISCUSSION TITLE
Monkey Metrics - Add customizable NPC's to add/remove

### DISCUSSION THREAD

**Discussion by itswatkins** [runelite/runelite#17237] [Ideas]
There are some NPC's that this plug in (notably the stacked npc's on a single tile) could benefit from adding, whether that's through just adding them directly (one that comes to mind is bloodvelds when bursting in catacombs) or if there's an added feature that let's us add/remove npcs by name, whichever is easier :)

**Comment by Nightfirecat** [comment]
We do not maintain the monkey metrics plugin, please direct this request to the plugin author.
